### PR TITLE
inspection for simple language combined with the others

### PR DESCRIPTION
**Pull request by davsclaus**

### CODE REVIEW

File: camel-idea-plugin/src/test/java/org/apache/camel/idea/inspection/CamelInspectJavaSimpleTestIT.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    public static final String CAMEL_CORE_MAVEN_ARTIFACT = "org.apache.camel:camel-core:2.19.0-SNAPSHOT";
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+        File[] mavenArtifacts = getMavenArtifacts(CAMEL_CORE_MAVEN_ARTIFACT);
+        VirtualFile virtualFile = LocalFileSystem.getInstance().refreshAndFindFileByIoFile(mavenArtifacts[0]);
+        final LibraryTable projectLibraryTable = LibraryTablesRegistrar.getInstance().getLibraryTable(myModule.getProject());
+        ApplicationManager.getApplication().runWriteAction(() -> {
```
Comment (fharms) [inline camel-idea-plugin/src/test/java/org/apache/camel/idea/inspection/CamelInspectJavaSimpleTestIT.java]: I found an easier why to add
 libscom.intellij.testFramework.PsiTestUtil#addLibrary

**Comment by fharms** [comment]
Nice work!

File: camel-idea-plugin/src/test/java/org/apache/camel/idea/inspection/CamelInspectJavaSimpleTestIT.java (hunk 1 above)
Comment (davsclaus) [inline camel-idea-plugin/src/test/java/org/apache/camel/idea/inspection/CamelInspectJavaSimpleTestIT.java]: Okay we can do a fix/PR with this for all the tests together

**Comment by fharms** [comment]
Sure, but let me do it in my pr then. I'm working on better testing of validation

**Pull request merged**